react/react_events/README.md

```diff
@@ -42,9 +44,9 @@ constructor() {
 }
 ```
 
-We first call `super()` to inherit the typical React component functionality.  
+We first call `super()` which calls the parent's constructor ( the class object constructor ) method to inherit the typical React component functionality.  
```

Comment:
See comment above. I think a bit more context earlier will make it clearer that `super()` means the `constructor()` function for `React.Component`